docusaurus.config.js

```diff
@@ -75,34 +75,29 @@ const config = {
                   to: "#",
                   logoClass: "panos",
                   docs: [
-                    {
-                      label: "APIs and SDKs",
-                      to: "panos/docs",
-                      icon: "doc"
-                    },
```

Comment:
Any reason this was left out? Can we add it back in?

Reply:
We opted to leave it out for now, since we aren't planning to migrate the existing content from panos.pan.dev. If you have _new_ content you'd like to include there we can add it back?